src/lib.rs

```diff
@@ -57,6 +57,8 @@ Serializing / Deserializing is as simple as calling `to_string` / `from_str`.
 
 !*/
 
+#![doc(html_root_url = "https://docs.rs/ron/5.1")]
```

Comment:
Please put a comment in `Cargo.toml` saying that this line needs to be adjusted accordingly